pennylane/transforms/optimization/optimization_utils.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions for circuit optimization."""
+
+from pennylane.math import allclose, sin, cos, arccos, arctan2, zeros, cast_like, stack
+from pennylane.wires import Wires
+
+
+def _find_next_gate(wires, op_list):
```

Comment:
since these functions are being imported elsewhere, it kinda feels like they _shouldn't_ be private 🤔 To me, private functions are generally used within the same module they are defined (not sure if there is a strong Python convention for this or not)